specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkManager.json

```diff
@@ -1,816 +0,0 @@
-{
```

Comment:
Why those existing files are deleted? should be just update the api-version?

Reply:
They are not existing files. 2022-06-01-preview is a new version, so far we have no any existing files. These files were copied from previous version when we create the branch

Reply:
Your new API version should include all existing APIs, so those files are needed. By deleting those files, you are deleting the existing APIs, so they won't exist in 2022-06-01-preview. Deleting APIs is not what you want.

Reply:
@mentat9, Thanks for your comments. Our new version will include all existing files, and we will use the files from stable version 2022-07-01. Please check https://github.com/Azure/azure-rest-api-specs/blob/19d617135fb1ea06e8e946b8e6711f8aa44740c4/specification/network/resource-manager/readme.md.

Reply:
@yanfa317 - Those other files are all declared with version 2022-07-01, not 2022-06-01-preview. Also, that readme file is only used for SDK generation, it is not part of the REST spec. If you want your REST spec to have those APIs with the correct API version, you need to include them as part of your 2022-06-01-preview spec.

Reply:
@mentat9, we only need new APIs are in version 2022-06-01-preview, and for all other services, they should still in version 2022-07-01. And we need all these existing services are included in our new generated SDK/PS packages. It is the way that we can make sure that we will get a full package of network, and the same time, we don't need to touch SDK/PS code for other services. Please check our old preview version (for example, 2022-04-preview). Thanks!